tetris.py

```diff
@@ -113,6 +115,7 @@ def __init__(self, canvas, width, height, outline_color, current_block_center, s
         self.small_board = small_board
         self.draw_board()
         self.draw_block()
+        self.moving_down = False
```

Comment:
Why is this in `Board`?

Reply:
It's a flag that determines whether the loop in user_input_down should be active or not. Functions move_block_down_press and move_block_down_release affect its state